### DISCUSSION TITLE
Multi-way keyboard and mouse sharing as opposed to server/client

### DISCUSSION THREAD

**Discussion by ghost** [input-leap/input-leap#1773] [Feature Requests]
> This issue has been migrated from old Barrier Github repository https://github.com/debauchee/barrier/issues/824
> 
> Issue created on: 2020-08-01 by @i2
> Issue last updated on: 2021-09-18

### Operating Systems ###

Server: Windows 7 version 6.1 (Build 7601, SP1)

Client: Arch Linux 5.7.11-arch1

### Barrier Version ###

2.3.3

### Question ###

Is there a plan to add two-way or multi-way mouse/keyboard sharing in Barrier as opposed to typical server/client method? (Just like what is implemented in Mouse-Without-Borders or Synergy)

---

> Commented on: 2020-08-02 by @shymega

No plans yet.

On this date - Sat, Aug 01, 2020 at 11:23:13AM -0700, Reza wrote:
> ### Operating Systems ###
>
> Server: Windows 7 version 6.1 (Build 7601, SP1)
>
> Client: Arch Linux 5.7.11-arch1
>
> ### Barrier Version ###
>
> 2.3.3
>
> ### Question ###
>
> Is there a plan to add two-way or multi-way mouse/keyboard sharing in Barrier as opposed to typical server/client method? (Just like what is implemented in Mouse-Without-Borders or Synergy)
>
>
> --
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly or view it on GitHub:
> https://github.com/debauchee/barrier/issues/824

--
Kind regards,
Dom Rodriguez (shymega)

---

> Commented on: 2021-02-18 by @PikminRed

I was excitedly ditching buggy Mouse Without Borders for Barrier, but ran into this in the install process and had to go back. It'd be nice to one day make the switch, as it's an invaluable feature for me, who has multiple computers I frequently walk between in my workspace.

---

> Commented on: 2021-05-19 by @gregor-anich-uibk

I would also appreciate this feature very much :-) Thanks for making barrier good and providing it for free!

---

> Commented on: 2021-09-18 by @thursdaybw

This issue should probably be labelled as a enhancement/feature request. Change it's focus on "is there plans" to "what are the plans". I'm curious and not hopefully how trivial an implementation like this would be.

**Comment by jrbgit** [comment]
This would be very useful for my workflow.
